src/main/java/net/education/kyivstar/config/CreateSchema.java

```diff
@@ -0,0 +1,59 @@
+package net.education.kyivstar.config;
+
+import java.sql.Connection;
+import java.sql.Statement;
+
+import static net.education.kyivstar.services.util.Utils.readSqlScriptFromFile;
+
+public class CreateSchema {
+    private DbConnector dbConnector;
+
+    public CreateSchema(DbConnector dbConnector) {
+        this.dbConnector = dbConnector;
+    }
+
+    public void createDataBase() {
+
+        try (Connection connection = dbConnector.connectMariaDb(false);
+             Statement statement = connection.createStatement()) {
```

Comment:
Add logging to indicate whether the database initialization was successful or not.